### DISCUSSION TITLE
Make mV an officially supported unit of voltage

### DISCUSSION THREAD

**Discussion by jbouwh** [home-assistant/architecture#851] [Entity Models] [answered]
This discussion is very similar to #847. The mV unit was already adopted by adding it to the `UnitOfElectricPotential` enum, but was never official supported.

https://github.com/home-assistant/core/pull/84805 was added to suggest adopting `mV` as an supported unit of voltage.

**Comment by frenck** [comment, marked as answer]
The `voltage` device class, including its single supported `V` unit of measurement), has been around longer than this architectural repository. Therefore, there is no possible history of why it was limited.

I could not find any other references in other places either (I tried to find the original PR that added it, with no success).

I've been discussing this (and #847) in multiple places on Discord with multiple (groups of) people. We all seem to agree that this addition is good to go.

Considering this popping up in the current beta (due to the new checks/warnings), we are even going to pull it into the current beta to reduce the number of reports, and to reduce the number of integration removing the device class because of it.

Not pulling it into beta now, would mean people would remove the device classes that later get supported again.

I'll pick up the PR linked above (and its sub-PRs) to ensure they land into the 2023.1 beta.

Thanks for the proposal @jbouwh  👍 

../Frenck
